internal/readpassword/trezor.go

```diff
@@ -5,22 +5,86 @@ import (
 
 	"github.com/rfjakob/gocryptfs/internal/exitcodes"
 	"github.com/rfjakob/gocryptfs/internal/tlog"
+
+	"github.com/xaionaro-go/cryptoWallet"
+	"github.com/xaionaro-go/cryptoWallet/vendors"
 )
 
-// Trezor reads 16 deterministically derived bytes from a
+// It's required initial values to be constant to get the resulting key to
+// be deterministic (see the comment above and within function "Trezor()").
+// This values could be just a bunch of zeros, but it seems to be a little
+// degenerate case that may add risks, so random values were used.
+//
+// This values were generated using command:
+//   dd if=/dev/random of=/dev/stdout bs=48 count=1 2>/dev/null | hexdump -e '8/1 "0x%02x, " "\n"'
+var (
+	TREZOR_DUMMYKEY = []byte{
+		0xfc, 0x3b, 0xe3, 0xa6, 0xe6, 0x61, 0x32, 0xbc,
+		0x95, 0x86, 0x79, 0x06, 0x70, 0xf9, 0x7c, 0x0a,
+		0xab, 0x05, 0x3b, 0x12, 0xff, 0x4e, 0xa8, 0x8b,
+		0x5b, 0x58, 0x0a, 0x8e, 0x42, 0xcf, 0x5e, 0x20,
+	}
+	TREZOR_NONCE = []byte{
+		0xc9, 0xf1, 0x6d, 0xab, 0xba, 0x16, 0x68, 0xc9,
+		0xcc, 0xb6, 0xb2, 0xcd, 0xbc, 0x4a, 0xb6, 0xcb,
+	}
+	TREZOR_KEY_NAME            = "gocryptfs"
+	TREZOR_KEY_DERIVATION_PATH = `m/10019'/0'/0'/0'`
```

Comment:
These can be `const`. Also, please follow Go naming like `trezorKeyName`.

There are a few places in gocryptfs that use  this style, like `PATH_MAX`, but this is only when there is Linux C define copied.

As you suggested, let's use `m/10019'/0'` for gocryptfs. I see no problem with you using `m/10019'/1'` and higher.